modules/troubleshooting-enabling-kdump-day-one.adoc

```diff
@@ -68,6 +68,19 @@ systemd:
 <4> If you want to change the contents of `/etc/sysconfig/kdump` from the default, include this section and modify the `inline` subsection accordingly.
 <5> For the `ppc64le` platform, replace `nr_cpus=1` with `maxcpus=1`, which is not supported on this platform.
 
+[NOTE]
+====
+To export the dumps to NFS targets, the `nfs` kernel module must be explicitly added to the configuration file:
++
+.Example `/etc/kdump.conf` file
+[source,text]
+----
+nfs penguin.example.com:/export/cores
```

Comment:
Penguin is cool but is there a convention we use for example servers in documentation? 
```suggestion
nfs penguin.example.com:/export/coreos
```

Reply:
we just use variations of `example.com`: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#node-names

Reply:
Maybe `server.example.com` ?

Reply:
Haha ! That's something I copied from RHEL NFS documentation :D

Reply:
OK I updated that.